src/styles/generic/_layout.scss

```diff
@@ -71,6 +71,13 @@ header {
 .function {
     flex: 10;
     margin: 20px 25px;
+  h2{
+      font-family: $main-font;
+  }
+  textarea {
+    font-size: 14px;
```

Comment:
This can be $main-font-size as well.

Reply:
Or inherit the font-family. Since you have specified the $main-font to be the font of the body